specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/InstanceFailoverGroups.json

```diff
@@ -385,6 +385,18 @@
       ],
       "type": "object",
       "properties": {
+        "secondaryType": {
```

Comment:
This is adding new property to existing api.
Need to create a new api-version for this change.

Reply:
this PR is committing to dev branch of the to-be-release API version: 2022-05-01-preview

Reply:
@NetRock please review this PR once again, REST API PR needs to merged ASAP. thanks